src/com/magento/idea/magento2plugin/actions/generation/dialog/NewCronGroupDialog.java

```diff
@@ -0,0 +1,266 @@
+    }
+
+    private void onAddScheduleGenerateEveryChanges() {
+        scheduleGenerateEvery.setEnabled(getAddScheduleGenerateEvery());
+    }
+
+    private void onAddScheduleAheadForChanges() {
+        scheduleAheadFor.setEnabled(getAddScheduleAheadFor());
+    }
+
+    private void onAddScheduleLifetimeChanges() {
+        scheduleLifetime.setEnabled(getAddScheduleLifetime());
+    }
+
+    private void onAddHistoryCleanupEveryChanges() {
+        historyCleanupEvery.setEnabled(getAddHistoryCleanupEvery());
+    }
+
+    private void onAddHistorySuccessLifetime() {
+        historySuccessLifetime.setEnabled(getAddHistorySuccessLifetime());
+    }
+
+    private void onAddHistoryFailureLifetime() {
+        historyFailureLifetime.setEnabled(getAddHistoryFailureLifetime());
+    }
+
+    private void onAddUseSeparateProcess() {
+        useSeparateProcess.setEnabled(getAddUseSeparateProcess());
+    }
+
+    private String getModuleName() {
+        return this.moduleName;
+    }
+
+    private String getCronGroupName() {
+        return cronGroupName.getText().trim();
+    }
+
+    private int getScheduleGenerateEvery() {
+        return (int) scheduleGenerateEvery.getValue();
+    }
+
+    private Boolean getAddScheduleGenerateEvery() {
+        return addScheduleGenerateEvery.isSelected();
+    }
+
+    private int getScheduleAheadFor() {
+        return (int) scheduleAheadFor.getValue();
+    }
+
+    private Boolean getAddScheduleAheadFor() {
+        return addScheduleAheadFor.isSelected();
+    }
+
+    private int getScheduleLifetime() {
+        return (int) scheduleLifetime.getValue();
+    }
+
+    private Boolean getAddScheduleLifetime() {
+        return addScheduleLifetime.isSelected();
+    }
+
+    private int getHistoryCleanupEvery() {
+        return (int) historyCleanupEvery.getValue();
+    }
+
+    private Boolean getAddHistoryCleanupEvery() {
+        return addHistoryCleanupEvery.isSelected();
+    }
+
+    private int getHistorySuccessLifetime() {
+        return (int) historySuccessLifetime.getValue();
+    }
+
+    private Boolean getAddHistorySuccessLifetime() {
+        return addHistorySuccessLifetime.isSelected();
+    }
+
+    private int getHistoryFailureLifetime() {
+        return (int) historyFailureLifetime.getValue();
+    }
+
+    private Boolean getAddHistoryFailureLifetime() {
+        return addHistoryFailureLifetime.isSelected();
+    }
+
+    private int getUseSeparateProcess() {
+        return useSeparateProcess.getSelectedItem().toString().equals("Yes") ? 1 : 0;
+    }
+
+    private Boolean getAddUseSeparateProcess() {
+        return addUseSeparateProcess.isSelected();
+    }
+
+    @SuppressWarnings({"PMD.UnusedPrivateMethod"})
+    private void createUIComponents() {
+        this.useSeparateProcess = new FilteredComboBox(getYesNoOptions());
+    }
+
+    private List<String> getYesNoOptions() {
+        return new ArrayList<>(Arrays.asList("Yes", "No"));
```

Comment:
Lets move these 2 values into common bundle. Please update everywhere where it's used.